components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -172,7 +172,7 @@ class GeckoEngineSession(
         }
 
         geckoSession.load(loader)
-        Fact(Component.BROWSER_ENGINE_GECKO_BETA, Action.IMPLEMENTATION_DETAIL, "GeckoSession.load").collect()
+        Fact(Component.BROWSER_ENGINE_GECKO_NIGHTLY, Action.IMPLEMENTATION_DETAIL, "GeckoSession.load").collect()
```

Comment:
@pocmo this seem a bit odd if this intentionally ? BROWSER_ENGINE_GECKO_**NIGHTLY**

Reply:
Argh, forgot about that. I'll update the PR.

Reply:
👍🏽